analyzers/src/SonarAnalyzer.CSharp/Extensions/IMethodSymbolExtensions.cs

```diff
@@ -27,22 +28,25 @@ namespace SonarAnalyzer.Extensions
     internal static class IMethodSymbolExtensions
     {
         // The signature of the Dispose method on IDisposable
-        internal static bool IsDisposeMethod(this IMethodSymbol symbol) =>
+        public static bool IsDisposeMethod(this IMethodSymbol symbol) =>
```

Comment:
* This could go into the `commons` project.
* It's a duplicate of `KnownMethods.IsIDisposableDispose`.
* It does not support explicit interface implementations.
* Is this method or `KnownMethods.IsIDisposableDispose` under test?

Reply:
That's an ugly rabbit hole. I've removed this method to address the duplication. It's only covered through the rule integration.